scripts/test.sh

```diff
@@ -45,6 +45,18 @@ if [[ ! ${TEST_TYPE} =~ "functional_" ]] && [[ ! ${TEST_TYPE} =~ "smoke_" ]] &&
   error "Unexpected <type>: ${TEST_TYPE}"
 fi
 
+if [[ ! ${CONFIGURATION} =~ "iphone-11pro" ]] && [[ ! ${CONFIGURATION} =~ "iphone-se-2" ]]; then
+  error "Unexpected <configuration>: ${CONFIGURATION}"
+fi
+
+if [[ ${CONFIGURATION} == "iphone-11pro" ]]; then
```

Comment:
[nit] Above you use "=\~" for comparison, but here "==". Suggest using "=\~" here.